packages/Stylers/src/includes.js

```diff
@@ -143,3 +144,30 @@ export const placeholders = css`
     opacity: 1;
   }
 `;
+
+export const WebComponentGlobalStyle = createGlobalStyle`
```

Comment:
This is a convenient way to expose this, great idea! 

We _could_ also expose a function that could be used within a `GlobalStyle` component in case the consumer wants to add other global styles at the same time... do you think it would be valuable to offer that flexibility? Or just wait until it's needed/requested?

Reply:
Also, it might be nice, for symmetry and completeness, to offer the equivalent for an app – a `<stylers.GlobalStyle>` or `<stylers.BodyGlobalStyle>`... what do you think?

Reply:
....aaaand what about the case where a web component is used with Sass, but _not_ with `styled-components`? Maybe we should also offer the Sass version of this?

Reply:
@mikrotron I thought we couldn't use Sass version as it will require to set up a compiler for it. But sure I can set up a sass version to keep our code consistent